### PR TITLE
Adding tree view wrappers

### PR DESCRIPTION
**Pull request by jameslkingsley**
I'll be adding more commits as I go, just here so people can see what's been done already.

### CODE REVIEW

**Comment by jameslkingsley** [comment]
That's all of them. Let me know if I've made any errors; most of the commands were practically identical.

**Comment by NouberNou** [comment]
Most of these look good, except colors can be passed using `sqf::color` like `const rv_color &`. It is a struct with `rgba` fields.

**Comment by jameslkingsley** [comment]
Okay I think that's right now.

**Comment by esteldunedain** [comment]
:+1:

File: src/client/intercept/client/sqf/uncategorized.cpp [hunk 1]
```diff
@@ -6531,5 +6531,234 @@ namespace intercept {
...
+
+            game_value params({
+                path,
+                value_
+            });
+
+            host::functions.invoke_raw_binary(client::__sqf::binary__tvsetvalue__control__array__ret__nothing, ctrl_, params);
+        }
+
+        void tv_sort(const control& ctrl_, const std::vector<int>& path_, const bool& reversed_ = false) {
```
Comment (esteldunedain) [inline src/client/intercept/client/sqf/uncategorized.cpp]: I think I heard the default parameters should ony be specificied on the .hpp file

File: src/client/intercept/client/sqf/uncategorized.cpp [hunk 2]
```diff
@@ -6531,5 +6531,234 @@ namespace intercept {
...
+
+            game_value params({
+                path,
+                reversed_
+            });
+
+            host::functions.invoke_raw_binary(client::__sqf::binary__tvsort__control__array__ret__nothing, ctrl_, params);
+        }
+
+        void tv_sort_by_value(const control& ctrl_, const std::vector<int>& path_, const bool& reversed_ = false) {
```
Comment (esteldunedain) [inline src/client/intercept/client/sqf/uncategorized.cpp]: Same situation as above

**Comment by Glowbal** [comment]
:+1: Thanks @jameslkingsley

**Pull request merged**